backend/Dockerfiles/Dockerfile.python3

```diff
@@ -18,11 +18,10 @@ ARG PACKAGING_VER
 RUN mkdir -p /src
 COPY ./libs/ /src/
 
-# trufflehog requires git
 # checkov requires awscli and libffi-dev
 RUN apk add -q --no-cache gcc musl-dev git make bash grep xz libffi-dev && \
```

Comment:
Should we update the `trufflehog requires git` comment to reflect what else in this Dockerfile needs it?